whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/NrtmQueryHandler.java

```diff
@@ -135,7 +132,7 @@ public void messageReceived(final ChannelHandlerContext ctx, final MessageEvent
             }
         }
 
-        channel.write(ChannelBuffers.EMPTY_BUFFER).addListener(ChannelFutureListener.CLOSE);
+        channel.writeAndFlush(Unpooled.EMPTY_BUFFER).addListener(ChannelFutureListener.CLOSE);
```

Comment:
Do we need to flush immediately?

Reply:
According to point 10 [here](https://netty.io/wiki/new-and-noteworthy-in-4.0.html#porting-the-server)

> 10. Replace ctx.write() with ctx.writeAndFlush()

And I noticed the system not working as it should when `ctx.write()` was used.